Pronoy/mobile slo widget (#11661)
DataDog/documentation: content/en/mobile/_index.md

## Widgets

### SLOs widget

{{< img src="mobile/slo_widget.png" alt="widget page" responsive="true" style="width:100%; background:none; border:none; box-shadow:none;">}}

View your [SLOs][13] from your mobile home screen with Datadog widgets. You can add any SLOs from your organization as a widget, along with a timeframe.

Timeframe options are:
- 7 days
- 30 days
- 90 days
- Previous week
- Previous month
- Week to date
- Month to date

> You can also choose which dashboard from your organization opens in the app when you tap on an SLO widget, allowing you to quickly dig deeper into your metrics.

Reviewer: ```suggestion
You can also choose a dashboard to open when you tap on an SLO widget, allowing you to quickly dig deeper into your metrics.
```

Author: updating copy to suggested change but adding a few clarifying words

> **Note**: If you do not choose a dashboard to open in the app when you tap on an SLO widget, the widget will simply open the app by default.

Reviewer: ```suggestion
**Note**: If you do not specify a dashboard to open, tapping the SLO widget opens the Datadog app.
```
